source/content/guides/drupal-9/drupal-9-unhosted/08-deploy-dev.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Migrate a Drupal 9 Site from Another Platform
```

Comment:
This page should go after uploading database and files

Reply:
Actually, it's a duplicate of 13-deploy.md so this one could be removed

Reply:
or maybe make it the same as 07-deploy-dev under drupal-9-unhosted-composer and keep it in its current place (before db and files upload)